utility/src/main/java/org/oppia/util/data/HtmlParser.kt

```diff
@@ -0,0 +1,38 @@
+package org.oppia.util.data
+
+import android.content.Context
+import android.text.Html
+import android.text.Spannable
+import android.widget.TextView
+
+private const val CUSTOM_TAG = "oppia-noninteractive-image"
```

Comment:
See my suggestions in #229 for naming these.

Reply:
Corrected variable names.